### ISSUE TITLE
rewrite or remove existing `fix` messages

### ISSUE THREAD

**Issue by a-frantz** [stjude-rust-labs/wdl#193]
Many of our `fix` messages are redundant, obvious, or otherwise unhelpful. We should use the `fix` feature more mindfully. That means supplying the "corrected" code or **adding** something to the diagnostic that isn't already conveyed. And if those can't be accomplished, the `fix` should be omitted.

related to #192

**Comment by a-frantz** [comment]
Copying some thoughts I typed up in slack:

Existing diagnostic:
```
note[BlankLinesBetweenElements]: missing blank line
   ┌─ tests/lints/blank-lines-between-elements/source.wdl:17:5
   │  
17 │ ╭     scatter (i in ["hello", "world"]) {
18 │ │         call bar { input: s = i }
19 │ │     }
   │ ╰─────^
   │  
   = fix: add a blank line before this element
```
We discussed being more intentional with the `fix` messages. So we probably want to drop this one as it doesn't add very much. That leaves:
```
note[BlankLinesBetweenElements]: missing blank line
   ┌─ tests/lints/blank-lines-between-elements/source.wdl:17:5
   │  
17 │ ╭     scatter (i in ["hello", "world"]) {
18 │ │         call bar { input: s = i }
19 │ │     }
   │ ╰─────^
```
Problem here is that now the Span doesn't make sense. Why are lines 18-19 included? Maybe the span's `len` should be `0`? That creates:
```
note[BlankLinesBetweenElements]: missing blank line
   ┌─ tests/lints/blank-lines-between-elements/source.wdl:17:5
   │
17 │     scatter (i in ["hello", "world"]) {
   │     ^
```
Ideally, I want an arrow that points _between lines 16 and 17_ and indicates _a blank goes here_. IDK how to do that in codespan. Can we do that?

Created by hand, but something like:
```note[BlankLinesBetweenElements]: missing blank line
   ┌─ tests/lints/blank-lines-between-elements/source.wdl:17:5
16 │     input {}
---->
17 │     scatter (i in ["hello", "world"]) {
```

(We can't do that in `codespan`)

The moral of this experiment is that reworking the `fix` messages may also require changing up some of the reported Spans to make more sense. That can be a bit tricky depending on how the rule is written. So just something to be mindful of.

**Comment by a-frantz** [comment]
It was decided to reverse the course of this issue and instead ensure that _every_ lint diagnostic has a `fix` included (#260 )
